app/jobs/create_new_person_job.rb

```diff
@@ -0,0 +1,14 @@
+class CreateNewPersonJob
+  include Sidekiq::Worker
+  sidekiq_options queue:'create_person'
```

Comment:
you need to add sidekiq.yml and add the queue name in

Reply:
agree